OasisDEX/oasis-borrow
# Added handling for max fee in gwei parameter

# [Added handling for max fee in gwei parameter](https://app.shortcut.com/oazo-apps/story/5014/max-gas-fee-handling-on-the-front-end-side-logic)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added handling for max fee in gwei parameter
  
## How to test 🧪
  <Please explain how to test your changes>

- try to setup different variants of gwei parameter both in basic sell and buy forms
- verify whether default state is loaded properly
- verify whether middle steps (editing state) work as expected
